.eleventy.js

```diff
@@ -61,11 +62,11 @@ module.exports = function(eleventyConfig, options={
           for( let pageNumber = 0, max = pagedItems.length; pageNumber < max; pageNumber++) {
             const currentNumber = pageNumber + 1
             tagMap.push({
-              slug: tagName,
+              slug: slugify(tagName),
               title: tagName,
               totalPages,
               posts: pagedItems[pageNumber],
```

Comment:
```suggestion
              [options.itemsCollection]: pagedItems[pageNumber],
```
If I'd retrieve data using the actual itemsCollection name, such as 'articles', for example.

```
{
    slug: 'dev',
    title: 'Dev',
    totalPages: 2,
    articles: [ [Object], [Object], [Object], [Object] ],
    permalinkScheme: 'dev/index.html',
    pages: { current: 1, next: 2, previous: false }
  }
```